frontstage/templates/surveys/surveys-todo.html

```diff
@@ -8,10 +8,10 @@
 
 {% include 'partials/tab_list.html' %}
 
-{% if just_added_case_id %}
+{% if added_survey %}
 <div class="panel panel--simple panel--success survey--success col-5@m u-mb-m ">
     <div class="panel__body">
-        <div>Your <a id="NEW_SURVEY_NOTIF" href="#ADDED_{{ just_added_case_id }}">new survey</a> has been added</div>
+        <div>Your new survey has been added</div>
```

Comment:
Any reason we decided to remove the ability to when you add a new survey having a link on it so it jumps to where on the list that new survey was added and highlighted?

Reply:
It was only highlighting one instance and multiple may have been added. Now it will highlight all but it can't jump to multiple places on the page. I am going to speak to Leah about this to see what she wants to do.